### DISCUSSION TITLE
A request to pair is not shown in a new g6 transmitter.

### DISCUSSION THREAD

**Discussion by tzachi-dar** [NightscoutFoundation/xDrip#2090] [Q&A] [answered]
Someone is trying to connect a new g6 transmitter, without success. In the logs it says "prompting user to note pairing..." but user says that he does not see any pairing request. How can this be solved?

![image](https://user-images.githubusercontent.com/7942372/165166371-8ece6ad5-e161-4c29-8005-85e8cc9b4ec9.png)
![image](https://user-images.githubusercontent.com/7942372/165166389-39c23f42-b5f3-4a2d-a11e-23896fafbe6e.png)
![image](https://user-images.githubusercontent.com/7942372/165166403-006e87fb-edb4-44e4-9a4d-abf1eb1a5f16.png)
![image](https://user-images.githubusercontent.com/7942372/165166413-2dde447b-cbea-43a6-b32b-d7e730598622.png)
![image](https://user-images.githubusercontent.com/7942372/165166432-9dfea2e5-3f71-4d2c-9b2c-080ab9f0dab9.png)

**Comment by Navid200** [comment, marked as answer]
This may or may not be it:
In Android, go to Settings -> Apps & notifications -> Notifications -> xDrip+

See if any of the xDrip notifications has accidentally been disabled.  If so, enable it.

**Comment by tzachi-dar** [comment]
Thanks Navid, it seems that this has helped.

Although after this change, the sound of notification was heard, but there was still no message.
She has pressed on this dialog:
![image](https://user-images.githubusercontent.com/7942372/165178840-d4a8f8ea-439d-4442-aaf3-9170688eb243.png)
The test there says: "Connect a new device"
A few minutes later the dexcom has connected.

**Reply by Navid200** [reply]
I'm glad it's working.

I have got that log on Android 8 and Android 9 as well.
However, I have never got a pair request on Android 8 or 9.  
I only get pair requests on my Android 11.  
My understanding is that a pair request is only shown on Android 10 or above.

So, it's hard to say what is the cause without having all the details.

**Comment by Navid200** [comment]
@jamorham Can I change this log to read "... on Android 10+ ..."?

![165166371-8ece6ad5-e161-4c29-8005-85e8cc9b4ec9](https://user-images.githubusercontent.com/51497406/165184785-7d2c5a1b-bf99-49d4-8af5-87f56dfa4f9c.png)

**Reply by tolot27** [reply]
> However, I have never got a pair request on Android 8 or 9.

I did get pairing requests on Android 8. Hence, I recommend not to change this text. Also, the pairing is slightly different on Android 8+ because the API has changed.

**Reply by Navid200** [reply]
Then, perhaps it is different by phone.  
I get a sound notification that the phone is now linked.  But, I never get a request for approval.

**Comment by tzachi-dar** [comment]
The phone on this specific case was Android 11.
